documentation/modules/auxiliary/scanner/dcerpc/endpoint_mapper.md

```diff
@@ -0,0 +1,130 @@
+## Description
+
+The endpoint_mapper module queries the EndPoint Mapper service of a remote system to determine what services are available. In the information gathering stage, this can provide some very valuable information.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/dcerpc/endpoint_mapper```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set THREADS [number of threads]```
+4. Do: ```run```
+
+In order to run the module, all we need to do is pass it a range of IP addresses, set the THREADS count, and let it go to work.
+
+## Scenarios
+
+**Running the scanner**
```

Comment:
Lets change this line to 

```
### Example Windows 2003, and Windows 7 Targets
```

Reply:
Hey @h00die, I am supposed to make those changes in my fork, correct?

Reply:
Yup. When you push the changes to your branch, the PR will automatically update